mystery_word.py

```diff
@@ -0,0 +1,154 @@
+import random
+
+
+def load_words(file_name):
+
+    with open(file_name) as file:
+        return (file.read().split())
+
+
+def pull_random_word(word_list):
+
+    return random.choice(word_list)
+
+
+def difficulty_choice():
+
+    user_input = input("[E]asy, [M]edium, or [H]ard: ").upper()
+
+    if user_input not in "EMH":
+        print("\n"
+              "Please type E, M, or H.")
+        difficulty_choice()
+    else:
+        return user_input
+
+
+def get_words(difficulty, word_list):
+
+    while True:
+        new_word = pull_random_word(word_list)
+
+        if difficulty == "E":
+            if len(new_word) > 6 or len(new_word) < 4:
+                continue
+            else:
+                break
+
+        elif difficulty == "M":
+
+            if len(new_word) > 10 or len(new_word) < 5:
+                continue
+            else:
+                break
+
+        elif difficulty == "H":
+
+            if len(new_word) < 10:
+                continue
+            else:
+                break
+
+    return new_word
+
+
+def play_again():
+
+    user_input = input("Would you like to play again? Y or N").upper()
+
+    if user_input not in "YN":
+        play_again()
+
+    return user_input
+
+
+def user_input_checker(guess):
+    # Clinton, for some reason including; if guess == "Quit": return False
+    # in the function didn't work to break out of my loop, so I had to include
+    # it outside of the function. can't figure out why
```

Comment:
Your loop was `while True` and return just returns from this function, so the loop keeps running.